Tests/DefaultIlpClientTest.swift

```diff
@@ -7,35 +7,90 @@ final class DefaultIlpClientTest: XCTestCase {
     // MARK: - Balance
     func testGetBalanceWithSuccess() {
         // GIVEN an IlpClient which will successfully return a balance from a mocked network call.
-        
+        let ilpClient = DefaultIlpClient(
+            balanceNetworkClient: FakeIlpBalanceNetworkClient.successfulFakeNetworkClient,
+            paymentNetworkClient: FakeIlpPaymentNetworkClient.successfulFakeNetworkClient
+        )
+
         // WHEN the balance is requested
-        
+        guard let balance: Org_Interledger_Stream_Proto_GetBalanceResponse =
+            try? ilpClient.getBalance(for: "foo", withAuthorization: "password") else {
+            XCTFail("Exception should not be thrown when trying to get a balance")
+            return
+        }
+
         // THEN the balance is correct
+        XCTAssertEqual(balance.accountID, "foo")
```

Comment:
Can you add a TODO for yourself to make these values in a shared location so we aren't relying on hardcoding two duplicate values.

Reply:
Did ya one better and wrapped `String` in a `typealias` for `AccountID` and `BearerToken`, and used your new `PaymentPointer` `typealias`, and also created some common test values